packages/components/src/circular-option-picker/circular-option-picker-option.tsx

```diff
@@ -66,7 +68,6 @@ function UnforwardedOptionAsOption(
 					ref={ forwardedRef }
 				/>
 			}
-			store={ compositeStore }
```

Comment:
This isn't needed because in theory `Composite.Item` should inherit the store from its parent `Composite`. The only scenario in which it would break, is if the option and its parent component are rendered across a slot / fill.